### PR TITLE
Node 8 Example

### PR DESCRIPTION
**Pull request by grayside**

### CODE REVIEW

**Comment by grayside** [comment]
* [x] Remove some dangling dullahan references
* [x] Update with simplifications/etc from retrofitting a project based on this PR

File: node8/docker-compose.override.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      # If you have issues with filesync and do not need file watches, please
+      # test the performance by replacing "dullahan-sync" with a period (.:/code)
+      - projectname-sync:/code
+      # Persist tool caches. This covers yarn and npm.
+      - /data/projectname/cache:/usr/local/share/.cache
+
+volumes:
+  projectname-sync:
+    external: true
+  dullahan-ssl-certs:
```
Comment (febbraro) [inline node8/docker-compose.override.yml]: should we rename this to `projectname-ssl-certs`?

Comment (febbraro) [reply node8/docker-compose.override.yml]: And, where does this volume come from? I dont see it defined anywhere and it is not labelled external, so I'm guessing it just gets created as part of this?

Comment (grayside) [reply node8/docker-compose.override.yml]: Yes, this is not an external volume. This is simply data persistence without a bind mount.

**Comment by febbraro** [comment]
You think this is getting closer to a merge?  Seems like a few dullahan removals are still in order?

**Comment by grayside** [comment]
sed -i .bak 's/dullahan/projectname/' node8/*

**Review by febbraro — APPROVED** [review]

**Pull request merged**